app/services/google/parser_service.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Google
+  class ParserService
```

Comment:
```suggestion
  class ParseResultsService
```

Same idea for using an action verb ;-)